packages/core/forms/src/forms/RLAForm.vue

```diff
@@ -0,0 +1,428 @@
+import { computed, ref } from 'vue'
+import english from '../locales/en.json'
+
+const EXAMPLES = [
+  {
+    description: 'A fixed limit of 500 requests per hour resetting sharply on the hour, ensuring no user can exceed this limit.',
+    config: {
+      limit: 500,
+      window_size: 3600,
+      window_type: 'fixed',
+    },
+  },
+  {
+    description: 'Users are allowed 200 requests per 30 minutes, resetting exactly on the 30 minute mark with no carryover of unused limits.',
+    config: {
+      limit: 200,
+      window_size: 1800,
+      window_type: 'fixed',
+    },
+  },
+  {
+    description: 'A strict limit of 5000 requests per day resetting promptly at midnight, preventing any burst traffic or inconsistent user experiences.',
+    config: {
+      limit: 500,
+      window_size: 86400,
+      window_type: 'fixed',
+    },
+  },
+  {
+    description: 'Maximum of 100 requests every rolling hour, continuously adjusting the count over the course of the hour. No hard limit or known reset.',
+    config: {
+      limit: 100,
+      window_size: 3600,
+      window_type: 'sliding',
+    },
+  },
+  {
+    description: 'Each user can make up to 300 requests in any rolling 30 minute period, with older requests dropping off as new requests are made.',
+    config: {
+      limit: 300,
+      window_size: 1800,
+      window_type: 'sliding',
+    },
+  },
+  {
+    description: 'Each user is allowed 500 requests per hour. \nIf the limit is exceeded and the user receives a 429 error, each additional request within the sliding window (hour) \nwill extend the wait time by about 12 minutes, continuously adjusting as new requests are made.',
+    config: {
+      limit: 500,
+      window_size: 3600,
+      window_type: 'sliding',
+      disable_penalty: false,
+    },
+  },
+]
+
+interface RequestLimit {
+  limit?: number
+  windowSize?: number
+}
+
+const { t } = createI18n<typeof english>('en-us', english)
+
+const props = defineProps<{
+  formSchema: any
+  formModel: Record<string, any>
+  formOptions: any
+  onModelUpdated: (value: any, model: string) => void
+  isEditing?: boolean
+}>()
+
+const scopingSchema = computed(() => {
+  const selectionGroup = props.formSchema?.fields?.find((field: any) => field.model === 'selectionGroup')
+  if (!selectionGroup) {
+    return undefined
+  }
+
+  return {
+    fields: [{
+      ...selectionGroup,
+      horizontalRadios: true,
+    }],
+  }
+})
+
+const advancedSchema = computed(() => {
+  const omittedFields = new Set([
+    'selectionGroup',
+    ...['identifier', 'limit', 'window_size', 'error_code', 'error_message']
+      .map((field) => `config-${field}`),
+  ])
+
+  return {
+    fields: props.formSchema?.fields
+      ?.filter((field: any) => typeof field.model === 'string' && !omittedFields.has(field.model))
+      ?.map((field: any) => {
+        if (field.model === 'config-redis-cluster_addresses') {
+          return {
+            ...field,
+            hint: 'e.g. localhost:6379',
```

Comment:
Should we internationalize this?

Reply:
@Leopoldthecoder No. I left it by mistake. Let me remove it.